app/src/main/java/de/westnordost/streetcomplete/settings/SettingsFragment.kt

```diff
@@ -100,6 +100,74 @@ class SettingsFragment : PreferenceFragmentCompat(), HasTitle,
         }
     }
 
+    private fun buildLanguageSelector() {
+        // FIXME: externalize to YML?
```

Comment:
Don't worry about this, I will create a gradle task after that creates/updates a list if this PR can be merged